.travis.yml

```diff
@@ -1,4 +1,7 @@
-language: python
+sudo: required
```

Comment:
Hmm. It used to work as recently as #22: https://travis-ci.org/phyloref/phylo2owl/jobs/147828629

I suppose what makes the difference is switching from `language: python` to `language java`. That switch causes `pip install` to fail without either `--user` using sudo? (I would not specify the version for `setuptools`. I vaguely recall that I ran into problems with that and then simply left it at whatever defaulted to the latest version.)

How about the alternative of leaving `language: python` and then installing Java on the fly? Does that not work because of the license restrictions on Java, which means you can't just download and install it (even if under a custom location and then setting `JAVA_HOME`)?